md/script/evalCodeBlocks.ts

```diff
@@ -56,15 +57,13 @@ export async function evalCodeBlocks(
     const iconLength = icon === "skip" ? 4 : 1;
     const langLength = String(lang).length;
 
+    const prefix = `${colorIcon} ${details.lang} `;
+    const spacesInPrefix = 2;
+
     const inlineCode = code.trim().replace(/\s+/g, " ");
-    const firstChars = inlineCode.slice(
-      0,
-      consoleWidth - langLength - iconLength - 5,
```

Comment:
`5` was used to account for 2 spaces in the non-`firstChars` portion of the `message` plus 3 periods at the end.

Now the non-`firstChars` portion of the `message` is extracted and given a name (`prefix`) and the well-named constant `spacesInPrefix` is used to explain what the number 2 signifies. The space for the ellipses is handled within `elideEnd()`.